### PR TITLE
Redirect to new accessbility URL

### PR DESCRIPTION
**Pull request by brandon-rusk**
Redirect https://cfpb.github.io/design-system/guidelines/accessibility
to
https://cfpb.github.io/design-system/guidelines/accessibility-principles

## Additions

- Added `gem 'jekyll-redirect-from'` plugin to redirect

## Testing

1. Put /design-system/guidelines/accessibility in URL and test if it redirects to new accessibility page

Resolves https://github.com/cfpb/design-system/issues/1324

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -2,3 +2,4 @@ source 'https://rubygems.org'
 gem "nokogiri", "~> 1.10"
 gem "jekyll", "~> 4.0"
 gem "jekyll-last-modified-at", "~> 1.1.0"
+gem 'jekyll-redirect-from'
```
Comment (chosak) [inline Gemfile:5]: Let's pin this like we do the other Ruby gems. This is [the latest version](https://rubygems.org/gems/jekyll-redirect-from) and also the version used [when you deploy](https://app.netlify.com/sites/cfpb-design-system/deploys/618018cecb45900007b794f8).

```suggestion
gem "jekyll-redirect-from", "~> 0.16.0"
```

**Review by chosak — APPROVED** [review]
Nice work! My one concern: if we manually add something to the Jekyll frontmatter, will it stick around if/when Netlify CMS rewrites the file? In other words, if we publish this, and then make an edit, will the line added manually get removed?

**Comment by anselmbradford** [comment]
> Nice work! My one concern: if we manually add something to the Jekyll frontmatter, will it stick around if/when Netlify CMS rewrites the file? In other words, if we publish this, and then make an edit, will the line added manually get removed?

How do we test this? Manually edit https://github.com/cfpb/design-system/blob/main/docs/pages/accessibility.md

**Comment by contolini** [comment]
> > Nice work! My one concern: if we manually add something to the Jekyll frontmatter, will it stick around if/when Netlify CMS rewrites the file? In other words, if we publish this, and then make an edit, will the line added manually get removed?
> 
> How do we test this? Manually edit https://github.com/cfpb/design-system/blob/main/docs/pages/accessibility.md and then edit the same page in the CMS? @contolini do you know the answer to this?

Yes, I just used these steps to test it locally and it rearranged the frontmatter fields but didn't delete anything. In the past when we've renamed cms fields we've had to manually delete all the old ones from the markdown files because netlify-cms never deletes anything, it just edits current YAML fields or adds new ones. So I think we're good-to-go! 👍

**Review by anselmbradford — APPROVED** [review]

**Pull request merged**